3634/0/triangle.rb

```diff
@@ -0,0 +1,36 @@
+# Triangle Project Code.
+# rubocop:disable Metrics/CyclomaticComplexity
+# rubocop:disable Metrics/AbcSize, Lint/UnneededCopDisableDirective
+# rubocop:disable Metrics/PerceivedComplexity
+# Triangle analyzes the lengths of the sides of a triangle
+# (represented by a, b and c) and returns the type of triangle.
+#
+# It returns:
+#   :equilateral  if all sides are equal
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+#
+def triangle(sd1, sd2, sd3)
+  raise TriangleError, 'Side must be > 0' if sd1 * sd2 * sd3 <= 0
+  raise TriangleError, 'Absurd' if sd1 + sd2 <= sd3 || sd1 + sd3 <= sd2 || sd2 + sd3 <= sd1
```

Comment:
![image](https://user-images.githubusercontent.com/2829832/60889629-23a4ac00-a262-11e9-8183-baa25d63b568.png)

Reply:
"Absurd" is latin word. Not russian.